src/main/kotlin/org/rust/lang/core/psi/ext/RsPath.kt

```diff
@@ -91,7 +91,7 @@ abstract class RsPathImplMixin : RsStubbedElementImpl<RsPathStub>,
 
     override val referenceNameElement: PsiElement
         get() = checkNotNull(identifier ?: self ?: `super` ?: cself ?: crate) {
-            "Path must contain identifier: $this ${this.text} at ${this.containingFile.virtualFile.path}"
+            "Path must contain identifier: $this ${this.text} at ${this.containingFile.virtualFile?.path}"
```

Comment:
Now `IllegalStateException` is thrown for every incomplete path (e.g. `Foo::`) in the editor.

Reply:
:+1: 
This is the reason why it's WIP